pkgs/top-level/all-packages.nix

```diff
@@ -37132,7 +37132,7 @@ with pkgs;
 
   lpairs2 = callPackage ../games/lgames/lpairs2 { };
 
-  ltris = callPackage ../games/lgames/ltris { };
+  ltris = lgames-ltris;
```

Comment:
Shouldn't this be in the aliases ?

Reply:
Aliases in `aliases.nix` are rejected by ofborg. I am not so sure if this is a good idea for now.

Reply:
The PR LGTM, but I'm totally unsure about this... maybe asking in Matrix? I don't really know what to do in here. I made a PR lately where I had to deal with aliases... see it by yourself, maybe it might give ideas? Here: https://github.com/NixOS/nixpkgs/pull/285828#issuecomment-1962887792